state-network.md

```diff
@@ -160,47 +160,92 @@ of proof payloads and to ensure that verification of proofs can be done in a
 single pass.
 
 
-#### Account Trie Node
+#### Account Trie Node for Offer
 
-This data type represents a node from the main state trie.
+This data type represents a node from the main state trie when offered via gossip.
 
 
 ```
 account_trie_node_key  := Container(path: Nibbles, node_hash: Bytes32)
 selector               := 0x20
 
-content_for_offer      := Container(proof: StateWitness, block_hash: Bytes32)
-content_for_retrieval  := Container(node: WitnessNode)
+content                := Container(proof: StateWitness, block_hash: Bytes32)
 content_key            := selector + SSZ.serialize(account_trie_node_key)
 content_id             := sha256(content_key)
 ```
 
-#### Contract Trie Node
+#### Account Trie Node for Retrival
 
-This data type represents a node from an individual contract storage trie.
+This data type represents a node from the main state trie when retrieved from another node by explicit request.
 
 
 ```
-storage_trie_node_key  := Container(address: Address, path: Nibbles, node_hash: Bytes32)
+account_trie_node_key  := Container(path: Nibbles, node_hash: Bytes32)
 selector               := 0x21
 
-content_for_offer      :=  Container(proof: StorageWitness, block_hash: Bytes32)
-content_for_retrieval  :=  Container(node: WitnessNode)
+content                := Container(node: WitnessNode)
+content_key            := selector + SSZ.serialize(account_trie_node_key)
+content_id             := sha256(content_key)
+```
+
+#### Contract Trie Node for Offer
+
+This data type represents a node from an individual contract storage trie when offered via gossip.
+
+
+```
+storage_trie_node_key  := Container(address: Address, path: Nibbles, node_hash: Bytes32)
+selector               := 0x22
+
+content                :=  Container(proof: StorageWitness, block_hash: Bytes32)
+content_key            :=  selector + SSZ.serialize(storage_trie_node_key)
+content_id             :=  sha256(content_key)
+```
+
+
+#### Contract Trie Node for Retrival
+
+This data type represents a node from an individual contract storage trie when retrieved from another node by explicit request.
```

Comment:
```suggestion
This data type represents a node from an individual contract storage trie when retrieved from another node via FINDCONTENT/FOUNDCONTENT.
```

Maybe these are better with explicit names for the message types?

Not going to make this suggestion for all of these (if you agree with this change it should be replicated to each of these definitions)